app/models/user.rb

```diff
@@ -1,2 +1,6 @@
 class User < ActiveRecord::Base
+  include Authem::User
+
+  validates_presence_of :email, :password
+  validates_uniqueness_of :email
```

Comment:
Model validations should most definitely have an accompanying spec. I would shy away from any libraries like shoulda however and just write the validation specs in a magic-free way using normal RSpec and ActiveRecord.